pkg/async/notifications/implementations/sandbox_publisher.go

```diff
@@ -0,0 +1,31 @@
+package implementations
+
+import (
+	"context"
+
+	"github.com/flyteorg/flytestdlib/logger"
+	"github.com/golang/protobuf/proto"
+)
+
+type SandboxPublisher struct{}
+
+var msgChan = make(chan []byte)
```

Comment:
can we instantiate this in the `NewSandboxPublisher()` constructor and make it a member variable? Should make things easier to unit test too!

Reply:
Alternatively we can pass it to NewSandboxPublisher() and NewSandboxProcessor()

Reply:
I think I can implement the alternative solution.
But how to pass the msgChan to first solution ?
Thanks a lot.

Reply:
Sure, you could do 

```
// Receiver only channel
func NewSandboxProcessor(msgChan <-chan []byte)  

// Sender only channel
and NewSandboxPublisher(msgChan chan<- []byte)
```

And continue to initialize e chan it outside of the processor and publisher

Reply:
Can I clarify what "continue to initialize msgChan" means?
I thought that I just have to initialize the msgChan 1 time only.